Docs.Site/Docs/Release-Notes/MyGet-2017.2.markdown

```diff
@@ -0,0 +1,123 @@
+# MyGet 2017.2 Release Notes
+
+MyGet 2017.2 was released on **TODO**, 2017.
+
+## Highlights
+
+Next to some new [features](#Features) and many [fixes](#Bug_Fixes_amp_Other_Improvements), this 2017.2 release of MyGet again adds some new functionality to the service.
+
+Major highlights of this release are: 
+
+* We added **PHP Composer** support, and welcome PHP developers to the MyGet family! ([Announcement](https://blog.myget.org/post/2017/09/11/php-composer-packages-on-myget.html) | [Docs](docs/walkthrough/getting-started-with-php-composer))
+
+<div style="text-align:center; display:block; margin-bottom:10px;">
+<a href="docs/walkthrough/getting-started-with-php-composer">
+	<img src="Images/myget-php-composer.svg" alt="Getting started with PHP Composer on MyGet!" width="90px"/>
+</a>
+</div>
+
+* MyGet is now also available in the **GitHub Marketplace**! ([Announcement](https://github.com/blog/2469-build-on-your-workflow-with-four-new-marketplace-apps))
+Full details, and the ability to set up a *free trial* from within GitHub, are available at [https://github.com/marketplace/myget](https://github.com/marketplace/myget).
+
+<div style="text-align:center; display:block; margin-bottom:10px;">
+<a href="https://github.com/marketplace/myget">
+	<img src="Images/gh-marketplace.png" alt="MyGet package management now available in the GitHub Marketplace!" width="400px"/>
+</a>
+</div>
+
+* In light of upcoming **EU General Data Protection Regulation** ([EU GDPR](https://www.eugdpr.org/)), which will be enforced in May 2018, MyGet is taking proactive action to verify we are compliant, and take corrective measures if we spotted anything that is not compliant, or questionable (typical grey area in legislation).
+Being a EU-based company, we take privacy and security very seriously! As such, we focused in this release to ensure that:
+
+	* user sign-ups by default opt-out of marketing communications or newsletters; unless the user explicitly takes positive action by ticking the checkbox to opt-in (which of course we do recommend, as we try to keep you informed about evolutions and guidance in the package management space)
+	* first-time visitors see a proper cookie consent banner, requiring so-called *double consent* from the user (in other words: no automatic consent which you may see on many other, non-compliant web sites using phrases like "by continuing to use this website...")
```

Comment:
`in other words: we ask you to explicitly accept/deny non-essential cookies instead of the automatic consent with cookies you see elsewhere on the Internet`